### DISCUSSION TITLE
Cannot accept or decline the first ever sent colaboration invitation

### DISCUSSION THREAD

**Discussion by Sifis-747** [community/community#135423] [New to GitHub]
Yesterday I received the above mentioned invitation in my smartphone and tried to click the "View invitation" button. I received an error message and now when I click on it in my laptop's browser it won't work. What should I do?

**Comment by geniuszly** [comment]
1. Sometimes, issues like this can be caused by a corrupted cache or cookies. Try clearing your browser's cache and cookies, then log back into GitHub and try accessing the invitation again.
2. If clearing the cache doesn’t work, try using a different web browser to see if the issue persists. For instance, if you’re using Chrome, try Firefox or Safari.
3. Ensure that your internet connection is stable. A weak or unstable connection can sometimes cause problems when trying to interact with web-based applications.
4. Sometimes browser extensions or settings can interfere with website functionality. Try opening GitHub in an incognito or private browsing window, where extensions are typically disabled, and see if that works.
5. GitHub might be experiencing issues. You can check [GitHub's status page](https://www.githubstatus.com/) to see if there are any ongoing problems that might be affecting the service.
6. If possible, ask the sender to resend the invitation. Sometimes, re-triggering the invitation can solve the problem.
7. If none of the above solutions work, you might need to contact GitHub Support. Provide them with details about the issue, including any error messages you received

**Reply by Sifis-747** [reply]
Dear @geniuszlyy,

Thank you for your prompt reaction! Considering it's Sunday, I was not expecting a response today. Thanks a million!

I tried all of the above and nothing worked. When I reached GitHub support I keep coming across the same screen where I'm asked to describe my issue (most probably by the AI) and couldn't find a way to actually reach out to someone for help. What I may have found out is that (maybe) my account has been "flagged" for some reason, but again didn't find a way to react.

Any ideas?

Thanks once again for your time!

**Comment by nitesh2920** [comment]
you can try the below steps

1. Clear Browser Cache and Cookies
      Clear your browser’s cache and cookies.
      Restart your browser and try accessing the invitation again.

2. Access Invitations Directly Through GitHub
    Log in to your GitHub account.
    Go to [GitHub Organization Settings](https://github.com/settings/organizations).
    Look for the "Organization invitations" section.
    Check for any pending invitations and attempt to accept or decline them.

3. Check Your Email for the Invitation Link
    If you received an email notification, use the link provided in the email to access the invitation.

**Reply by Sifis-747** [reply]
1. I tried to clear both browser's and computer's cach by disabling/reenabling the ethernet adapter
2. I couldn't find any "Organization invitations" section
3. the e-mail I rfeceived contains the following link: https://github.com/me50/Sifis-747/invitations but when it connects to GitHub it changes to https://github.com/me50/Sifis-747. I tried to complete it manually by adding "/invitations" but it didn't work

**Reply by nitesh2920** [reply]
2. click on the link [GitHub Organization Settings](https://github.com/settings/organizations) you will redirect to that section.
3. if it is not working then maybe they have cancelled the invitation.

**Reply by Sifis-747** [reply]
Thank you for your time and good will to offer your help!

The invitation arrived yesterday and, as it's mentioned in the mail, it is valid for 7 days. Why should they cancel it?

**Reply by nitesh2920** [reply]
I mean that by mistake may be some error happened you can ask to resend it.

**Reply by Sifis-747** [reply]
Thank you once again for your time!

I received the message from noreply@github.com which discouraged me from sending a reply. However, I've got nothing to lose, so I'll follow your advice.

Thanks a lot!

**Reply by Sifis-747** [reply]
As expected, my message was blocked with the following message: Message bounced due to organizational settings.

**Comment by TheGiraffe3** [comment]
Are you logged in in your web browser? That may be the problem.

**Comment by Sifis-747** [comment]
I have checked and I am logged in properly.

Now I can notice that your answer is not visible on GitHub!

Kind regards,
Sifis-747
